app/src/main/java/com/example/bisit/ui/home/HomeListAdapter.kt

```
@@ -31,11 +31,19 @@ class HomeListAdapter(
 
         holder.binding.tvName.text = item.shopName
         holder.binding.tvCategory.text = item.category
-        holder.binding.tvRating.text = item.averageRating.toString()
+        val formattedRating = String.format("%.1f", item.averageRating)
```

detekt (1.23.8)

[warning] String.format("%.1f", item.averageRating) uses implicitly default locale for string formatting.

(detekt.potential-bugs.ImplicitDefaultLocale)